daiquiri/formatter.py

```diff
@@ -23,21 +25,138 @@
 )
 
 
-class ColorFormatter(logging.Formatter):
-    # TODO(jd) Allow configuration
-    LEVEL_COLORS = {
-        logging.DEBUG: '\033[00;32m',  # GREEN
-        logging.INFO: '\033[00;36m',  # CYAN
-        logging.WARN: '\033[01;33m',  # BOLD YELLOW
-        logging.ERROR: '\033[01;31m',  # BOLD RED
-        logging.CRITICAL: '\033[01;31m',  # BOLD RED
-    }
+class Palette(object):
```

Comment:
Why not simply pass level colors as an argument of the constructor with a default value that it the current/old `LEVEL_COLORS`?